EsperBenchmark/FilterBenchmark/src/main/java/DataGenerator.java

```diff
@@ -1,4 +1,5 @@
 public class DataGenerator {
```

Comment:
Lets add the copyright header for all the java source files.

Reply:
Added copyright header